src/Acl.php

```diff
@@ -23,11 +22,16 @@ class Acl
     public $auth;
 
     /**
-     * Returns AccessRules model for logged in user and in model scope.
+     * @internal
+     */
+    private bool $skipApplyRestrictionsForRulesExport = false;
+
+    /**
+     * Returns array of AccessRules records for logged in user and in particular model scope.
      *
-     * @return AccessRule
+     * @return list<array{model: string, all_visible: bool, visible_fields: list<string>, all_editable: bool, editable_fields: list<string>, all_actions: bool, actions: list<string>, conditions: list<string>}>
```

Comment:
```suggestion
     * @return list<array{model: class-string<Model>, all_visible: bool, visible_fields: list<string>, all_editable: bool, editable_fields: list<string>, all_actions: bool, actions: list<string>, conditions: list<string>}>
```

is `model` strictly `class-string<Model>`?

how the `conditions` strings look like?

Reply:
`conditions` string is not implemented yet. I will probably be serialized (maybe json) array of conditions, but will see.

Reply:
if so, I prefer to not export it at all

and what about the `model`, is my assumption right?